### PR TITLE
Remove not required jobs

### PR DESCRIPTION
**Pull request by lego963**
Refers to: #577 and #588

### CODE REVIEW

**Comment by gtema** [comment]
I would say it's not temporary, in #577 we should drop them as such and migrate docs

File: .travis.yml [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+  # See: https://github.com/golang/go/issues/12933
+  - bash scripts/gogetcookie.sh
 
 script:
-- make test
-- make vet
-- make website-test
+  - make test
+  - make vet
+#- make website-test
```
Comment (outcatcher) [inline .travis.yml]: Please remove it completely. Don't comment out unused things

File: .zuul.yaml [hunk 2]
```diff
@@ -14,13 +14,15 @@
       jobs:
         - golang-make-test
         - golang-make-vet
-        - golang-make-website-test
+        #- golang-make-website-test
```
Comment (outcatcher) [inline .zuul.yaml]: See prev comment

File: .zuul.yaml [hunk 3]
```diff
@@ -14,13 +14,15 @@
...
+        #NEED TO BE FIXED: in #577, #588
         - goreleaser-build
     gate:
       jobs:
         - golang-make-test
         - golang-make-vet
-        - golang-make-website-test
+        #- golang-make-website-test
```
Comment (outcatcher) [inline .zuul.yaml]: See prev comment

**Review by outcatcher — CHANGES_REQUESTED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**